src/platforms/stm32/src/lib/sys.c

```diff
@@ -17,20 +17,25 @@
  *
  * SPDX-License-Identifier: Apache-2.0 OR LGPL-2.1-or-later
  */
+#include <time.h>
 
 #include <avmpack.h>
 #include <defaultatoms.h>
+#include <interop.h>
 #include <scheduler.h>
 #include <sys.h>
 
+// #define ENABLE_TRACE
 #include <trace.h>
 
 #include "avm_log.h"
 #include "gpiodriver.h"
+#include "stm_sys.h"
 
 #define TAG "sys"
 
-void sys_tick_handler();
```

Comment:
let's avoid noise, let's do this kind of housekeeping with other PRs.

Reply:
Yes, that was not intended here. It is a leftover I missed when preparing this PR. That fix is included in #814.